src/rgw/sts-assume-role.cc

```diff
@@ -225,6 +227,47 @@ std::tuple<int, RGWRole> STSService::getRoleInfo(const string& arn)
   }
 }
 
+int STSService::_storeARNandPolicy(string& policy, string& arn)
```

Comment:
why leading '_' in method name?

Reply:
Because this is a private method, can remove it.